client/landing/stepper/declarative-flow/flows/site-migration-flow/site-migration-flow.ts

```diff
@@ -403,15 +402,8 @@ const siteMigration: Flow = {
 				}
 
 				case STEPS.SITE_MIGRATION_INSTRUCTIONS.slug: {
-					return navigate(
-						addQueryArgs(
-							{
-								siteId,
-								siteSlug,
-								from: fromQueryParam,
-							},
-							STEPS.SITE_MIGRATION_STARTED.slug
-						)
+					return exitFlow(
```

Comment:
Could you please try this approach?
...
useStepNavigation() {
....
const { reset } = useDispatch( SITE_STORE );
...
}


```suggestion
	             reset();
				return exitFlow(
```

It should reset the site store, forcing the overview page to load the site info.

Reply:
I was trying the following locally, and I think it's not working because location change is meaning that we don't seem to process the dispatch call.
```suggestion
					if ( siteId ) {
						const removeSiteFromStateAction = {
							type: SITE_DELETE_RECEIVE,
							siteId: siteId,
						};

						// Note that I added a `const dispatch = useDispatch();` call in useStepNavigation()
						dispatch( removeSiteFromStateAction );
					}
					return exitFlow(
```